pkg/dash/dash.go

```diff
@@ -542,13 +544,23 @@ func enableOpenCensus() error {
 	return nil
 }
 
-// findKubeConfig looks for kube config from .kube or provided by user
-func findKubeConfig(logger log.Logger, kubeConfig string) error {
-	if _, err := os.Stat(kubeConfig); err == nil {
-		logger.Infof("using kube config: %v", kubeConfig)
-		return nil
+// validateKubeConfig returns a valid file list of kube config(s)
+func validateKubeConfig(logger log.Logger, kubeConfig string) (string, error) {
```

Comment:
Let's add a basic test for this method, should be easy to verify the behavior of using a bad path and a good path.

Reply:
Switched to afero instead of `os.Stat` for testing purposes